qiskit_nature/problems/sampling/protein_folding/__init__.py

```diff
@@ -14,14 +14,17 @@
 =================================================================================
 The protein to be folded is defined in a Peptide class. Each peptide consists of one and only one
 main chain and optionally several side chains. Side chains cannot be attached to first, second or
```

Comment:
```suggestion
main chain and optionally several side chains. Side chains cannot be attached to the first, second or
```
This will likely break `style` and `lint`...